package.json

```diff
@@ -19,7 +19,7 @@
     "prepare": "husky install",
     "prettier": "prettier --cache --ignore-path .prettierignore --write --list-different .",
     "release": "pnpm build && changeset publish",
-    "test": "jest",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest",
```

Comment:
For anyone reviewing, this is needed for the miniflare test environment - https://miniflare.dev/testing/jest
It doesn't cause any breaking changes in this project

Reply:
@enisdenjo Isn't this that was causing memory leaks ?

Reply:
it causes Jest's LeakDetector to misbehave. since we're not using `--detectLeaks` here, we should be ok

Reply:
Thank you for the insight :-)